src/ui.js

```diff
@@ -582,6 +584,9 @@ HanabiCard.prototype.add_listeners = function() {
 	            self.note_given.show();
 	        } else {
 	            self.note_given.hide();
+	            self.tooltip.hide();
```

Comment:
If there's an appropriate object to have a function that clears these, I'll gladly refactor it.